src/components/Modal/ConfirmModalLayout.tsx

```diff
@@ -0,0 +1,58 @@
+import Button from "../Button";
+
+import CheckIcon from "@/assets/icon/check.svg?react";
+import NoticeIcon from "@/assets/icon/notice.svg?react";
```

Comment:
```suggestion
import { Check, Notice } from "@/assets/icon";
``` 
수빈님이 export 해주셔서 위처럼 사용할 수 있습니다!👍